package.json

```diff
@@ -7,14 +7,16 @@
     "dev": "vite",
     "build": "vite build",
     "lint": "eslint src --ext js,jsx --report-unused-disable-directives --max-warnings 0",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "format": "prettier --write \"**/*.+(js|jsx|json|css|md)\""
   },
   "dependencies": {
     "axios": "^1.4.0",
     "dayjs": "^1.11.8",
     "jwt-decode": "^3.1.2",
     "jwt-encode": "^1.0.1",
     "miragejs": "^0.1.47",
+    "prettier": "^2.8.8",
```

Comment:
Are you aware of the differences between dependencies and dev-dependencies? If not, read on it and then let's decide whether we want eslint and prettier to be,

Reply:
Yes, devDependencies are required for developing the application and dependencies for running the application.